certified-connectors/Spinpanel/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "1d2b61d3-a040-4c69-b597-b20aad4dd0b0",
```

Comment:
@VincentMejan Please remove the client id value as per Microsoft's security policy. You can replace it with dummy text.
`"clientId": "[[DUMMY]]"`

Reply:
Replaced with [[DUMMY]]